From 4ff4bf0625dc11f8d2e216399742523d2a80758e Mon Sep 17 00:00:00 2001
From: Adam Williamson <awilliam@redhat.com>
Date: Thu, 3 Jan 2013 12:43:09 -0800
Subject: [PATCH 121/126] French Canadian xlayout is just "ca", not "ca(fr)"
 any more (cherry picked from commit fa593cfd8783374ec1382f32ea02f4a137a6de16)

---
 src/locale/kbd-model-map | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/locale/kbd-model-map b/src/locale/kbd-model-map
index a0d7d19..e05322d 100644
--- a/src/locale/kbd-model-map
+++ b/src/locale/kbd-model-map
@@ -66,7 +66,7 @@ sk-qwerty		sk	pc105		-		terminate:ctrl_alt_bksp,qwerty
 dev			dev,us	pc105		-		terminate:ctrl_alt_bksp,grp:shifts_toggle,grp_led:scroll
 fr-latin9		fr	pc105		latin9		terminate:ctrl_alt_bksp
 fr_CH-latin1		ch	pc105		fr		terminate:ctrl_alt_bksp
-cf			ca(fr)	pc105		-		terminate:ctrl_alt_bksp
+cf			ca	pc105		-		terminate:ctrl_alt_bksp
 sv-latin1		se	pc105		-		terminate:ctrl_alt_bksp
 sr-cy			rs	pc105		-		terminate:ctrl_alt_bksp
 gr			gr,us	pc105		-		terminate:ctrl_alt_bksp,grp:shifts_toggle,grp_led:scroll
-- 
1.8.1

